src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/config-detail.jelly

```diff
@@ -2,27 +2,47 @@
 <j:jelly xmlns:j="jelly:core" xmlns:c="/lib/credentials"
          xmlns:f="/lib/form"
          xmlns:scm="/jenkins/scm/api/form">
-  <j:choose>
-    <j:when test="${descriptor.apiUriSelectable}">
-    <f:entry title="${%API endpoint}" field="apiUri">
-      <f:select/>
-    </f:entry>
-    </j:when>
-    <j:otherwise>
-      <f:invisibleEntry>
-        <f:select field="apiUri"/>
-      </f:invisibleEntry>
-    </j:otherwise>
-  </j:choose>
-  <f:entry title="${%Credentials}" field="credentialsId">
-    <c:select checkMethod="post"/>
-  </f:entry>
-  <f:entry title="${%Owner}" field="repoOwner">
-    <f:textbox/>
-  </f:entry>
-  <f:entry title="${%Repository}" field="repository">
-    <f:select/>
-  </f:entry>
+  <f:radioBlock name="url" value="Github Repository" checked="${instance.isURLRepo() == ''}" title="Github Repository Scan" inline="true">
```

Comment:
`isURLRepo` should return `boolean` rather than `String`.

```suggestion
  <f:radioBlock name="url" value="Github Repository" checked="${instance.isURLRepo()}" title="Github Repository Scan" inline="true">
```

I might also rename the method to something like `isConfiguredByURL`.

Reply:
I makes more sense tbh, but I was following some examples found